ca/server.go

```diff
@@ -60,7 +62,7 @@ type Server struct {
 	// the security config as a result
 	lastSeenClusterRootCA *api.RootCA
 	lastSeenExternalCAs   []*api.ExternalCA
-	secConfigMu           sync.Mutex
+	signingMu             sync.Mutex
```

Comment:
I'm not sure `signingMu` is the right name. It seems to protect `s.externalCA`, and some aspects of the `securityConfig`.

Let's try to articulate what this mutex is used for, and from that we can figure out if the name makes sense.